ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -245,19 +248,18 @@ export const InteractiveNotebook = fp.flow(withUrlParams(), withCurrentWorkspace
               <div style={{display: 'flex'}}>
                 <div style={
                   Object.assign({}, styles.navBarItem,
-                    this.canWrite() ? styles.clickable : styles.disabled)}
+                    this.canWrite ? styles.clickable : styles.disabled)}
```

Comment:
this bit of logic can also be a property like `this.theButtonStyle` so you don't have to repeat it on line 259.

Reply:
Will add.